component/class/defaults.yml

```diff
@@ -15,5 +15,12 @@ parameters:
         repository: 'vshn/exoscale-metrics-collector'
         tag: 'v0.3.0'
     # Times in UTC! Don't run job around midnight as exoscale API may return incomplete data
+
+    # schedule for objectstorage cronjob
     # default: Every day at minute 10 past hour 10, 16 and 20.
     schedule: '10 10,16,20 * * *'
```

Comment:
Nitpick:  In such a case I'd probably deprecate the `schedule` parameter and add a `objectstorageschedule` or similar.  

You can see an example where we did something like this without a breaking change: https://github.com/projectsyn/component-registry-cache/blob/7cf8cacb1fa38f468651540849d52ac61bd7e546/component/registry.jsonnet#L14-L22

But again it's also fine as is

Reply:
done :+1: